h2/src/test/org/h2/test/db/TestFunctions.java

```diff
@@ -104,6 +104,7 @@ public void test() throws Exception {
         testSource();
         testDynamicArgumentAndReturn();
         testUUID();
+        testBase64();
```

Comment:
We prefer pure SQL test cases, see `TestScript` and `h2/src/test/org/h2/test/scripts/functions/string`.